### ISSUE TITLE
[3rd] Tradução Cap. 4 - "Data Structures: Objects and Arrays"

### ISSUE THREAD

**Issue by joaostein** [braziljs/eloquente-javascript#403]
Responsável: @osergioneto

**Comment by joaostein** [comment]
@osergioneto, estou enviando essa mensagem para todos os responsáveis apenas para checar se está tudo certo com a sua tradução.  Se precisar de qualquer coisa, só avisar.

_ps. A data da [milestone](https://github.com/braziljs/eloquente-javascript/milestone/2) para finalizar a tradução está para 24 de Dezembro (ho-ho 🎅🎁).  Se você achar que não for realista, só avisar, ok?_

**Comment by joaostein** [comment]
@osergioneto - poderia dar um update e qual o seu plano para tradução deste capítulo?

**Comment by cristofersousa** [comment]
Pessoal, queria ajudar na tradução desse capítulo, rola ainda?

**Comment by joaostein** [comment]
@cristofersousa claro, como @osergioneto está ausente, passei esse capítulo para você.

Para atualizar o _status_ dessa _issue_, você poderia, por gentileza, me dar um ping por aqui quando iniciar a tradução?

**Comment by cristofersousa** [comment]
@joaostein, vou tentar iniciar já nesse final de semana. 
Obrigado pela oportunidade =)

**Comment by joaostein** [comment]
Encerrando — pulamos a 3ª edição e fomos direto pra 4ª (#490).
